<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frameExtractHelpDlg</class>
 <widget class="QDialog" name="frameExtractHelpDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Frame numbers are extracted from the image file name.  Fo rthis to work they have to be in the same location for each file.  This window helps with setting the indices for the frame number within the name. Please copy and paste the full image name (with or wihtout extension) into the box below.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="filenameEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Now put the starting and ending values (inclusive) into the two boxes below.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="startInd"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="endInd"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="extractBtn">
     <property name="text">
      <string>Extract frame number</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resultLabel">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
